examples/debian_snapshot/BUILD.bazel

```diff
@@ -54,66 +54,52 @@ cacerts(
     }),
 )
 
-PACKAGES = [
-    "@bullseye//ncurses-base",
-    "@bullseye//libncurses6",
-    "@bullseye//tzdata",
-    "@bullseye_nolock//bash",
-    "@bullseye//coreutils",
-    "@bullseye//dpkg",
-    "@bullseye//apt",
-    "@bullseye//perl",
-    "@bullseye//openssl",
-    "@bullseye//nvidia-kernel-common",
-]
-
-# Creates /var/lib/dpkg/status with installed package information.
-dpkg_status(
-    name = "dpkg_status",
-    controls = select({
-        "@platforms//cpu:arm64": [
-            "%s/arm64:control" % package
-            for package in PACKAGES
-        ],
-        "@platforms//cpu:x86_64": [
-            "%s/amd64:control" % package
-            for package in PACKAGES
-        ],
-    }),
-)
-
 oci_image(
-    name = "apt",
-    architecture = select({
-        "@platforms//cpu:arm64": "arm64",
-        "@platforms//cpu:x86_64": "amd64",
-    }),
+    name = "image",
+    architecture = "arm64",
```

Comment:
Shouldn't this be reverted so that `architecture` is still a `select` so that we build and test two architectures?

Or is that done via the `platform_transition` and this is just a placeholder? :-?

Reply:
Yes, this is just metadata, platform_transition_filegroup transitions to arm64 always.

Reply:
I've checked and it seems that this is not correct. The resulting image has the architecture always set to `arm64` even when the image is built for `amd64` :-/